bin/cli.js

```diff
@@ -35,10 +35,16 @@ prettyCLI
         desc: 'Theme path points to a zip file'
     })
     .boolean('-1, --v1', {
-        desc: 'Check theme for Ghost 1.0 compatibility, instead of 2.0'
+        desc: 'Check theme for Ghost 1.0 compatibility'
+    })
+    .boolean('-2, --v2', {
```

Comment:
Another approach to specifying the version could be something like: `gscan --version 2` or `gscan --v 2` or some other name instead of `version` as it's reserved for current script's version check. Don't like this approach as it's more typing and breaks the existing convention.

Reply:
I agree. the wording looks good to me